instances/infrastructure-committee.near/widget/components/rfps/CommentsAndLogs.jsx

```diff
@@ -147,18 +147,11 @@ const Comment = ({ commentItem }) => {
     blockHeight,
   };
   const content = JSON.parse(Social.get(item.path, blockHeight) ?? "null");
-  const link = getLinkUsingCurrentGateway(
-    `${REPL_INFRASTRUCTURE_COMMITTEE}/widget/app?page=rfp&id=${props.id}&accountId=${accountId}&blockHeight=${blockHeight}`
-  );
-  function getHighlightCommentStyle() {
-    const highlightComment =
-      parseInt(props.blockHeight ?? "") === blockHeight &&
-      props.accountId === accountId;
-
-    return {
-      border: highlightComment ? "2px solid black" : "",
-    };
-  }
+  const link = `https://${REPL_INFRASTRUCTURE_COMMITTEE}.page/rfp/${props.id}?accountId=${accountId}&blockHeight=${blockHeight}`;
```

Comment:
@petersalomonsen what's the correct URL for displaying an rfp comment with scroll? like we have for proposal.

Reply:
That would be the correct URL you have there, but the contract also needs to support it here: https://github.com/NEAR-DevHub/neardevhub-contract/blob/main/src/web4/handler.rs#L127 ( currently it only supports proposal as you can see, not rfp )

Reply:
Here's a PR for adding support in the contract: https://github.com/NEAR-DevHub/neardevhub-contract/pull/152